runtime/src/bank.rs

```diff
@@ -11362,13 +11362,21 @@ pub(crate) mod tests {
             genesis_config,
             mint_keypair,
             ..
-        } = create_genesis_config_with_leader(500, &solana_sdk::pubkey::new_rand(), 1);
+        } = create_genesis_config_with_leader(10_000_000_000, &solana_sdk::pubkey::new_rand(), 1);
         let bank = Arc::new(Bank::new_for_tests(&genesis_config));
 
         let stake_delegations = bank.stakes_cache.stakes().stake_delegations().clone();
         assert_eq!(stake_delegations.len(), 1); // bootstrap validator has
                                                 // to have a stake delegation
 
+        let balance = {
+            let rent = &bank.rent_collector().rent;
```

Comment:
I got had by this while updating the RPC test -- could you double-check that there's non-zero rent here? https://github.com/solana-labs/solana/pull/24670/files#diff-6507431a1db5f52ddbedfa0242280da09cca66214f3ca42e0aba4387e6b1d3e7 has the fix

Reply:
Yah, by default rent is free. Updated to use the default Rent. Done!